scripts/send-submissions-to-prepare-for-export/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+  echo "The number of parameters is not correct. You need to provide the following: ./run.sh ipContainerPrepareSubmissionsForExport"
+  exit 1
+fi
+
+CONTAINER_IP=$1
+
+while IFS="" read -r l || [ -n "$l" ]
+do
+  curl --location --request POST $CONTAINER_IP/delta \
```

Comment:
Would it be better to have a one-liner to directly fetch the container IP? This way we don't have to manually do it and pass it to the script:
```suggestion
  curl --location --request POST "$(docker inspect "$(docker compose ps -q prepare-submissions-for-export)" --format '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}')"/delta \
```